### DISCUSSION TITLE
Adding the ability to add heading to the flux command.items group

### DISCUSSION THREAD

**Discussion by Ashraam** [livewire/flux#669] [Feature Requests]
Hi,

Everything is in the title, in the command component you can have multiple command.items, it would be great to be able to add a heading to them in order to separate them.

I thing it could be useful for multiple index searching for example.

Thanks

**Comment by jeffchown** [comment]
@Ashraam As Flux is 'composable' (https://fluxui.dev/docs/principles#composition), have you tried adding a `flux:heading` inside your `<flux:command.items>`? e.g.:
(I tested it and it worked)

```
<flux:heading class="!text-gray-400 ml-10 pt-4">COMMAND GROUP HEADING</flux:heading>
```
